### ISSUE TITLE
Python version

### ISSUE THREAD

**Issue by robhazes** [ariaaan/traffic-signs-detector#16]
Hi, great project!

I would like to test it and read that you used Python3. What was the exact version number of Python? I would like to run it in exactly the same configuration to avoid version-problems. I did not find out for sure looking trough the requirements.txt file.

Best regards, Rob.

**Comment by ariangilesgarcia** [comment]
Hi Bob! Thank you for reaching out.

I don't think there would be a problem with the python version, but I think I used `python3.5` if I can recall. The main problem is with the `tensorflow` and other requirements version, because it's a little outdated. I'm trying to get it updated but I haven't get the change yet.

If you have any other question don't hesitate to write me.

Have a good day!

**Comment by robhazes** [comment]
Thanks for your quick reply :) I am now working on installing this, and all the requirements installed without any problem except for tensorflow.

This is the Pyton version I went for: Python 3.5.4rc1 (64-bit)
I upgraded pip to version 20.0.2.

When I try to install tensorflow version 1.4.1 I get this message, see below.

Any suggestion on what to try next? 

-------------------------------------------------
ERROR: Could not find a version that satisfies the requirement tensorflow==1.4.1 (from versions: 0.12.0rc0, 0.12.0rc1, 0.12.0, 0.12.1, 1.0.0, 1.0.1, 1.1.0rc0, 1.1.0rc1, 1.1.0rc2, 1.1.0, 1.2.0rc0, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.2.1, 1.3.0rc0, 1.3.0rc1, 1.3.0rc2, 1.3.0, 1.4.0rc0, 1.4.0rc1, 1.4.0, 1.5.0rc0, 1.5.0rc1, 1.5.0, 1.5.1, 1.6.0rc0, 1.6.0rc1, 1.6.0, 1.7.0rc0, 1.7.0rc1, 1.7.0, 1.7.1, 1.8.0rc0, 1.8.0rc1, 1.8.0, 1.9.0rc0, 1.9.0rc1, 1.9.0rc2, 1.9.0, 1.10.0rc0, 1.10.0rc1, 1.10.0, 1.11.0rc0, 1.11.0rc1, 1.11.0rc2, 1.11.0, 1.12.0rc0, 1.12.0rc1, 1.12.0rc2, 1.12.0, 1.12.2, 1.12.3, 1.13.0rc0, 1.13.0rc1, 1.13.0rc2, 1.13.1, 1.13.2, 1.14.0rc0, 1.14.0rc1, 1.14.0, 1.15.0rc0, 1.15.0rc1, 1.15.0rc2, 1.15.0rc3, 1.15.0, 1.15.2, 2.0.0a0, 2.0.0b0, 2.0.0b1, 2.0.0rc0, 2.0.0rc1, 2.0.0rc2, 2.0.0, 2.0.1, 2.1.0rc0, 2.1.0rc1, 2.1.0rc2, 2.1.0, 2.2.0rc0, 2.2.0rc1, 2.2.0rc2, 2.2.0rc3)
ERROR: No matching distribution found for tensorflow==1.4.1

----------------------------------------------------------

**Comment by ariangilesgarcia** [comment]
Tensorflow is needed for `darkflow`: https://github.com/thtrieu/darkflow
Darkflow repo says it needs `tensorflow` -> `v1.0`
Maybe try using 1.0.0 or something like that.

**Comment by robhazes** [comment]
Great, that worked. Should I try to install the newest version possible between 1.0.0 and 1.4.1?

**Comment by robhazes** [comment]
Some more feedback:
When I open the example_image.py file in Pycharm I see the following message before I run the file:
Package requirement 'tensorflow==1.4.1' is not satisfied 

Ignoring this and running the file it exits with the following erros:

C:\Users\Rob\AppData\Local\Programs\Python\Python35\python.exe C:/traffic-signs-detector-master/example_image.py
Traceback (most recent call last):
  File "C:/traffic-signs-detector-master/example_image.py", line 2, in <module>
    from detector.detector import create_detector_from_file
  File "C:\traffic-signs-detector-master\detector\detector.py", line 13, in <module>
    from detector.localizer import Localizer
  File "C:\traffic-signs-detector-master\detector\localizer.py", line 1, in <module>
    from darkflow.net.build import TFNet
ImportError: No module named 'darkflow'

Process finished with exit code 1

Your advice is higly appreciated :)

**Comment by alevikpes** [comment]
@robhazes , there is an [automatic PR about the TF version](https://github.com/ariaaan/traffic-signs-detector/pull/12)
Try to apply it to your local

**Comment by robhazes** [comment]
Just wanted to let you know that with very good help from [https://github.com/alevikpes](url) we got it working :) I uploaded my version now including the large files. [https://github.com/robhazes/traffic-signs-detector](url)

**Comment by ariangilesgarcia** [comment]
That's so cool! Sorry I couldn't help you guys earlier.
I'll take a look at it to see what was wrong.

Thank you! :D

**Comment by mnby323** [comment]
Hi, would you mind to share with me how the yolo model is been trained towards the data? Im planning to train on my own data. Is there any specific tutorial that you have been referred to?

**Comment by robhazes** [comment]
Hi,

I managed to set up a training system using this guide. 
https://www.youtube.com/watch?v=Rgpfk6eYxJA

My goal is to make the trained model run on a raspberry, as shown here:
https://www.youtube.com/watch?v=aimSGOAUI8Y
I got his working too with the supplied example trained model.

I am however struggeling with findiing images for training with annotation files in the correct format, so at the moment I have not been training a model yet.

But these examples could help you.

Would be nice to hear when you succeed…

Best regards, Rob.
